### ISSUE TITLE
feat: Custom OG images for each page

### ISSUE THREAD

**Issue by kempsterrrr** [Developer-DAO/academy-turbo#217]
#215 introduces a default meta image for every page across the site. While this is a significant improvement over no meta images, we still need custom meta images for each page. This is important because sharing links to lessons and tracks can deliver a much better UX. Examples below.

**Current Default image**

The image below appears on every page. In this image, we provide minimal information to the user.

![default-meta-image](https://github.com/Developer-DAO/academy-turbo/assets/9025997/9f4d1024-4972-4c9b-b8ef-6787b6d27851)

**Example of improved image**
This shows the user more info in the image about what they're clicking and provides some variability in the branding aligned with the platform's internal branding. This looks better in the feed and will make a noticeable difference to the number of people who click these links, thus improving the academy's impact.

![intro-to-eth-example](https://github.com/Developer-DAO/academy-turbo/assets/9025997/0f5c6ddf-260b-48d6-ae30-09db2f1091ff)

**How do we do this?**

1. Automate the creation of this image via [Vercel's of image generation lib](https://vercel.com/docs/functions/og-image-generation)
2. Create them all manually

The work to automate this is not that large, so I recommend we do that as it will reduce workload as the platform scales.

**Comment by kempsterrrr** [comment]
This issue closes #214 once #215 is merged into master

**Comment by kempsterrrr** [comment]
Closing this issue as solve manually for now, we can open another in the future if we want to do this programmatically
